docs/next.config.js

```diff
@@ -46,10 +46,13 @@ if (reactStrictMode) {
   console.log(`Using React.StrictMode.`);
 }
 
+const isDeployment = process.env.NETLIFY === 'true';
+
 module.exports = withTM({
   eslint: {
     ignoreDuringBuilds: true,
   },
+  assetPrefix: isDeployment ? '/toolpad' : '',
```

Comment:
A comment could be good.

```suggestion
  // Avoid conflicts with the other Next.js apps hosted under https://mui.com/
  assetPrefix: isDeployment ? '/toolpad' : '',
```